src/models/Transaction.js

```diff
@@ -15,6 +15,8 @@ export const DIRECTION = {
   outgoing: 'outgoing'
 }
 
+export const FAILED_TRANSACTION_MESSAGE = 'The recipient for the transaction could not be found.  The transaction will appear as declined'
```

Comment:
Are we ending these messages with punctuation?  If so needs a period.

Reply:
Currently, we haven't been, but let's circle back with Rob and C on this one, if it's an interest.  Otherwise, we're at least remaining consistent. :+1: